tests/testsuite.src/run_file.at

```diff
@@ -12429,6 +12429,241 @@ AT_CHECK([diff reference prog.out], [0], [], [])
 AT_CLEANUP
 
 
+AT_SETUP([INDEXED file manipulation under ASCII/EBCDIC collation])
+AT_KEYWORDS([runfile WRITE DELETE READ EBCDIC])
+
+AT_SKIP_IF([test "$COB_HAS_ISAM" = "no"])
```

Comment:
Change to `AT_SKIP_IF([test "$COB_HAS_ISAM" != "db"])` and add a comment that this is only included for the BDB backend so far.